data/templates/default.typst

```diff
@@ -69,8 +72,26 @@ $endif$
 $if(fontsize)$
   fontsize: $fontsize$,
 $endif$
+$if(mathfont)$
+  mathfont: ($for(mathfont)$"$mathfont$",$endfor$),
+$endif$
+$if(codefont)$
+  codefont: ($for(codefont)$"$codefont$",$endfor$),
+$endif$
```

Comment:
Curious why you used `$for$` here.  `mathfont` would normally be expected to be a string in pandoc templates.